src/main/features/offer/routes/response.ts

```diff
@@ -0,0 +1,56 @@
+import * as express from 'express'
+import { FormValidator } from 'forms/validation/formValidator'
+import { Paths } from 'offer/paths'
+import { Form } from 'forms/form'
+import { DefendantResponse } from 'offer/form/models/defendantResponse'
+import { ErrorHandling } from 'common/errorHandling'
+import User from 'idam/user'
+import { OfferGuard } from 'offer/guards/offerGuard'
+
+function getPartyStatement (claim: any): any[] {
+  if (!claim.settlement || !claim.settlement.partyStatements) {
+    throw Error('Something went wrong, No offer present')
```

Comment:
This is not desired solution. Having this would mean that when citizen hits the page URI at a time there is no offer they will get 500 error. We wanna handle this scenario more gracefully so the possible solutions are:

- having business error page saying offer has not been made by defendant yet or more generic one saying an action is not available at this stage (preferred)
- redirect silently to dashboard without error message

Reply:
the use can only see this page if they have offer, if they don't the link is usually disabled. This if only safeguards against directly entering url in the browser.